test('throws invalid number', async () => {
 // Do nothing
});

